api/src/main/java/org/openmrs/module/dbevent/listener/EventModulePublisher.java

```diff
@@ -0,0 +1,63 @@
+package org.openmrs.module.dbevent.listener;
+
+import org.openmrs.event.Event;
+import org.openmrs.event.EventMessage;
+import org.openmrs.module.dbevent.DbEvent;
+import org.openmrs.module.dbevent.DbEventListener;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Objects;
+
+/**
+ * DbEventListener which uses the Event module to publish each DbEvent within the Event module
+ * for consumption by JMS Event Listeners
+ */
```

Comment:
This is another example / POC that demonstrates how we could potentially connect changes from Debezium to the Event module, so that downstream consumers in OpenMRS that are currently subscribing to, say, Patient object changes could simply make a switch to instead start subscribing to "patient" table changes.  Obviously it's not that simple since to replicate "Patient" you'd need to watch multiple tables, but this is a starting point for discussion.